src/views/Create.vue

```diff
@@ -68,13 +68,13 @@
           </div>
           <div class="column-sm hide-sm">
             <div
-              v-text="_num(price.values[token], 'currency')"
+              v-text="_num(price.values[token], $t('currency'))"
```

Comment:
Same here, it shouldn't be translated everywhere on `_num()` function
```suggestion
              v-text="_num(price.values[token], 'currency')"
```